src/coloris.js

```diff
@@ -62,7 +62,14 @@
           }
           break;
         case 'theme':
-          picker.className = `clr-picker clr-${options.theme.split('-').join(' clr-')}`;
+          if (['large', 'polaroid'].includes(options.theme)) {
+            classNames.push(options.theme);
+          }
```

Comment:
It's not necessary to check the value of the theme option here as it will become messy if more themes are added.